backend/reservations/ingest_models.py

```diff
@@ -0,0 +1,69 @@
+"""Datatypes for importing guest and room lists"""
+# we don't need a StaffImport model because that table is directly translated to the db table
+
+from typing import Optional
+from pydantic import BaseModel, Field, ValidationError
+
+class SecretPartyGuestIngest(BaseModel):
+    """Required fields imported from SecretParty CSV
+    """
+    ticket_code: str  # transaction code for purchase/transfer
+    last_name: str
+    first_name: str
+    email: str
+    product: str  # product code, eg name of addon for event
+    transferred_from_code: Optional[str] = None
+    transferred_from_email: Optional[str] = None
+    invitation_code: Optional[str] = None
```

Comment:
Is invitation code used anywhere, or planned? type as well? 
Looking at that thread in slack @otakup0pe also requested `transferred_from_name` not sure if we still want that. 
I would prolly lean towards less is better. open tho. non-blocking merge

Reply:
`transferred_from_name` could be maybe useful for some orphan detection / data corruption addressing but it's not critical. Reasonably sure we will never need `invitation_code` in roombot.

Reply:
leaving them in, because it is part of the current export from SP